extensions/binding/org.eclipse.smarthome.binding.lifx/ESH-INF/config/config.xml

```diff
@@ -27,4 +27,15 @@
         </parameter>
     </config-description>
 
+    <config-description uri="channel-type:lifx:color">
+        <parameter name="powerOnBrightness" type="integer" min="0" max="100" required="false" unit="%">
+            <label>Power on brightness</label>
+            <description>Brightness level used when switching on the light. Use empty value to leave brightness as is.</description>
+        </parameter>
+        <parameter name="powerOnColor" type="text" pattern="(36[0]|3[0-5][0-9]|[12][0-9][0-9]|[1-9][0-9]|[0-9]),\s*(100|[1-9][0-9]|[0-9]),\s*(100|[1-9][0-9]|[0-9]),\s*([0-9]|[1-9][0-9]|[1-9][0-9][0-9]|[1-8][0-9][0-9][0-9]|9000)" required="false">
+            <label>Power on Color</label>
+            <description>Comma separated Hue (0-360), saturation (0-100), brightness (0-100) and kelvin (0-9000) levels used when switching on the light. Use empty value to leave it as is.</description>
```

Comment:
You are absolutely right about the kelvins. Didn't think much, just did ;). I modified it so that both percent values and kelvins are supported for temperature. Personally I like the kelvins more than percent. 

Color channels support also "power on brightness" option, so user which just want to keep color but change the brightness can still use it.

Reply:
>  I modified it so that both percent values and kelvins are supported for temperature.

But you are still configuring the behavior of the color temperature channel in the color channel?

Reply:
How about now?

Reply:
Yes people with whites only lights will be grateful for this @paulianttila! :smile: 

You might also want to add the new English strings to the `i18n/lifx.properties` file so people know what has to be translated. If you're fluent in Dutch you can also update those translations. :wink:

Reply:
> You might also want to add the new English strings to the i18n/lifx.properties file

done